docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -138,17 +138,17 @@ BAU parameter data tables
    :widths: 30, 20, 10, 10
    :header-rows: 1
 
-.. csv-table:: Probability of testing LDL-c 
-   :file: prob_testing_ldlc.csv
-   :widths: 20, 10, 10
+.. csv-table:: Table 4: Probability of Adherence (Table 4a, 4b) parameters
+   :file: adherence_parameters.csv
+   :widths: 30, 20, 10, 10
    :header-rows: 1
 
-.. csv-table:: Adherence parameters
-   :file: adherence_parameters.csv
+.. csv-table:: Table 6: Current treatment practice - distribution by drug type 
+   :file: current_rx.csv
    :widths: 30, 20, 10, 10
    :header-rows: 1
 
-.. csv-table:: Probability of adverse events
+.. csv-table:: Table 7: Probability of side effect (adverse events)
    :file: prob_adverse_events.csv
```

Comment:
If a patient experiences a side effect, they will be given a different drug on their next visit.  The treatment algorithm assumes these patients are not adherent.